src/otx/core/utils/utils.py

```diff
@@ -84,3 +85,10 @@ def get_idx_list_per_classes(dm_dataset: DmDataset, use_string_label: bool = Fal
     for k in stats:
         stats[k] = list(dict.fromkeys(stats[k]))
     return stats
+
+
+def get_obj_from_str(obj_path: str) -> Any:  # noqa: ANN401
+    """Get object from import format string."""
+    module_name, obj_name = obj_path.rsplit(".", 1)
+    module = importlib.import_module(module_name)
+    return getattr(module, obj_name)
```

Comment:
Function name seems not telling what it does; `import_object_from_module()`?

Reply:
I see. I'll make it more readable

Reply:
done https://github.com/openvinotoolkit/training_extensions/pull/3759/commits/589c4c57264a27a5da2cfd9bb9aa3b2788c53bef

Reply:
done https://github.com/openvinotoolkit/training_extensions/pull/3759/commits/589c4c57264a27a5da2cfd9bb9aa3b2788c53bef